### PR TITLE
add GitHub workflow that sets up a React Native project with the library

### PR DESCRIPTION
**Pull request by koenpunt**
I've updated the UI Tests in the iOS project to assert an ad being rendered, and setup up GitHub CI to run these tests.

### CODE REVIEW

**Comment by koenpunt** [comment]
Hey @sbugert, can you maybe sign up for the GitHub Actions beta, so I can use it in this repo? 

https://github.com/features/actions

Signup should be immediately accepted.

File: .github/workflows/fixtures/App.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { AdMobBanner } from 'react-native-admob';
+
+const App = () => {
+  return (
+    <Fragment>
+      <StatusBar barStyle="dark-content" />
+      <SafeAreaView>
+        <ScrollView
+          contentInsetAdjustmentBehavior="automatic"
+          style={styles.scrollView}
```
Comment (sbugert) [inline .github/workflows/fixtures/App.js]: `styles.scrollView` is not defined because there is no StyleSheet

**Comment by koenpunt** [comment]
I was thinking similar things, but when I pushed it to my fork, it just worked; https://github.com/koenpunt/react-native-admob/runs/233747514

So not sure what the issue is, maybe they are still gradually updating users from the old format to the yaml format...

File: .github/workflows/fixtures/App.js (hunk 1 above)
Comment (koenpunt) [reply .github/workflows/fixtures/App.js]: Good catch!

**Comment by koenpunt** [comment]
> Btw great idea

Yeah I'm quite done with users saying "it doesn't work", while for me it works just fine. So I figured if I can demonstrate it's working on CI, I can simply refer to that :)

**Comment by sbugert** [comment]
> Btw, I'm seeing this banner in the repo; 
> So maybe you can change something in the settings of the repo?

I don't see a banner but I can click on the actions tab. But it only let's me create a new workflow there. And I did not find any settings related to actions.

Does it already work correctly for your fork? Then, I guess there's no reason not to try merging it to master and see if it works then. Or we could just wait to see if they're really still migrating the users :)

**Comment by koenpunt** [comment]
I've finalised the config, and it's running correctly on my fork: https://github.com/koenpunt/react-native-admob/commit/250daa4dc296d27e0072fe9a8cfddf6d9dddca9c/checks

**Comment by sbugert** [comment]
Looks great! I can see it now in the actions tab. Maybe because it's not a draft PR anymore?

Running it every 7 days sounds good.
The only reason for it to break is Google changing something, isn't it?

**Comment by koenpunt** [comment]
> The only reason for it to break is Google changing something, isn't it?

That, or the React Native implementation changes. But both are good to know early on of course.

**Pull request merged**